docs-markdown/src/test/suite/controllers/cleanup/cleanup-controller.test.ts

```diff
@@ -23,12 +23,16 @@ const filePath = resolve(__dirname, "../../../../../../src/test/data/repo/articl
 const folderPath = resolve(__dirname, "../../../../../../src/test/data/repo/articles");
 
 suite("Cleanup Controller", () => {
+    suiteSetup(async () => {
```

Comment:
I don't see any tests that are impacted by from stubbing `sendTelemetryData`.  We  also stub and restore this method 20+ times.  This change creates a single stub which is created once before all tests and restored once when all tests are completed.  It also prevents the tests from throwing the following stack trace during runtime

> stack trace: TypeError: Cannot read property 'sendTelemetryEvent' of undefined
> 	at Object.sendTelemetryData (/Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/helper/telemetry.js:41:26)
> 	at Object.<anonymous> (/Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:162:29)
> 	at Generator.next (<anonymous>)
> 	at /Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:8:71
> 	at new Promise (<anonymous>)
> 	at /Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:4:12
> 	at /Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:67:53
> 	at new Promise (<anonymous>)
> 	at /Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:67:20
> 	at Object._withProgress (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:544:963)
> 	at Object.withProgress (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:544:717)
> 	at Object.withProgress (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:867:255)
> 	at Object.<anonymous> (/Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:55:25)
> 	at Generator.next (<anonymous>)
> 	at fulfilled (/Users/mevans/mygit/vscode-docs-authoring/docs-markdown/out/src/controllers/cleanup/cleanup-controller.js:5:58)
> extensionHostProcess.js:920